styleguide-app/Examples/Button.elm

```diff
@@ -78,6 +79,16 @@ update msg state =
             in
             ( state, Cmd.none )
 
+        ToggleToggleButton id ->
+            ( { state | pressedToggleButtons =
+                if Set.member id state.pressedToggleButtons then
+                    Set.remove id state.pressedToggleButtons
+                else
+                    Set.insert id state.pressedToggleButtons
```

Comment:
I could have sworn there was a helper for this in Set.Extra or Sort.Set but nope.

Reply:
Oh wait there is! this is totally fine as is though. https://package.elm-lang.org/packages/stoeffel/set-extra/latest/Set-Extra#toggle